sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -550,6 +548,49 @@ pi_result _pi_device::getAvailableCommandList(
   return pi_result;
 }
 
+void _pi_queue::adjustBatchSizeForFullBatch() {
+  // QueueBatchSize of 0 means never allow batching.
```

Comment:
NIT: I think it can't happen in today's code the QueueBatchSize is equal to 0

Reply:
That is incorrect.  In the queue itself, QueueBatchSize of 0 means - No Batching.  Because I need to allow QueueBatchSize to be 1 when dynamic batching is being used, because the possibility is that dynamic batching can adjust QueueBatchSize down to as low as 1 if we are still doing too many partial batches.  So, in the queue itself, QueueBatchSize == 0 means no batching, QueueBatchSize > 0 is just the current bacthing size, and the queue bool UseDynamicBatching controls whether dynamic batch adjustment ever changes QueueBatchSize up or down.

Reply:
> So, in the queue itself, QueueBatchSize == 0 means no batching

I realized this, yeah. But the code (in _pi_queue constructor) seems to never set it to 0.

Reply:
On looking at the code in pi_level_zero.hpp, I see that you are correct.  Batching turned off is now really
represented as QueueBatchSize == 1 and UseDynamicBatching== false, or to say it another way, batching is turned off by using a fixed batch size of 1.  Thank you for pointing that out.